persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/JdbcBasePersistenceImpl.java

```diff
@@ -427,43 +428,28 @@ public Page<EntityNameLookupRecord> listEntities(
       long catalogId,
       long parentId,
       @Nonnull PolarisEntityType entityType,
+      @Nonnull PolarisEntitySubType entitySubType,
       @Nonnull PageToken pageToken) {
+    // TODO: only fetch the properties required for creating an EntityNameLookupRecord
```

Comment:
To help us remember as well, can we add `and update the underlying CONSTRAINT clause to make the implicit index INCLUDE the subTypeCode` to this TODO?

Reply:
i'll take a stab at this as an immediate follow-up or file it as a separate issue where i will take note of this additional "index coverage" requirement

Reply:
Related `dev` ML discussion: https://lists.apache.org/thread/kxyqp287zmxmorod7l205ojkwfqgrhkv